### PR TITLE
Blog post: Import AWS AMIs as KubeVirt Golden Images

### PR DESCRIPTION
**Pull request by davidvossel**
This blog post outlines the basics behind how to import AWS AMIs into a kubevirt cluster and execute kubevirt VMs using those imported AMIs.

### CODE REVIEW

File: _posts/2021-09-21-Importing-EC2-to-KubeVirt.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+## Nuts and Bolts of Importing AMIs
+
+### Official AWS AMI Export Support
+
+AWS supports an [api](https://docs.aws.amazon.com/vm-import/latest/userguide/vmexport_image.html) for exporting AMIs as a file to an s3 bucket. This support works quite well, however there's a long list of [limitations](https://docs.aws.amazon.com/vm-import/latest/userguide/vmexport_image.html#limits-image-export) that impact what AMIs are eligible for export. The most limiting of those items is the one that prevents any image built from an AMI on the marketplace from being eligible for the official export support.
+
+### Unofficial AWS export Support
+
+Regardless of what AWS officially supports or not, there's absolutely nothing preventing someone from exporting an AMI's contents themselves. The technique just involves starting creating an EC2 instance, attaching an EBS volume (containing the AMI contents) as a block device, then streaming that block devices contents where ever you want.
```
Comment (jobbler) [inline _posts/2021-09-21-Importing-EC2-to-KubeVirt.md:43]: Th **starting creating** in the following line is hard to read, I think one of those should be removed.:
The technique just involves **starting creating** an EC2 instance,

Comment (davidvossel) [reply _posts/2021-09-21-Importing-EC2-to-KubeVirt.md]: ha, fixed. thanks for pointing that out

**Pull request merged**